src/content/self-hosted/guides/okta/index.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Use Okta as your Authentication Provider
+description: Guide on how use Okta as the authentication provider of your Okteto Self-hosted instance
+id: identity-provider-okta
```

Comment:
You are missing adding it to the sidebar, or is it on purpose until we have the right section?

Reply:
it's on purpose for now